docs/rbe/rbe.md

```diff
@@ -0,0 +1,161 @@
+# RBE for Flutter Engine Developers
+
+## Overview
+
+This is documentation on setting up RBE for building the Flutter engine. It is
+expected to work for Googlers only on corp Linux, macOS, and Windows hosts,
+including cloudtop instances.
+
+## Getting started
+
+The first step is to add an entry to the `.gclient` file. The entry to add is
+`"use_rbe": True` in the `custom_vars` section. It should look like this:
```

Comment:
Drive by comment, as I'm following these docs at the moment (because my understanding is goma no longer works?) 

What should it look like? Should there be more after `:`

Reply:
Maybe like 
```
solutions = [
  {
    "managed": False,
    "name": "src/flutter",
    "url": "git@github.com:gmackall/engine.git",
    "custom_deps": {},
    "deps_file": "DEPS",
    "safesync_url": "",
    "custom_vars": {
      "use_rbe": True,
    },
  },
]
```